contributor-guide/modules/ROOT/pages/build-systems/b2.adoc

```diff
@@ -1,5 +1,33 @@
 = B2
+:navtitle: B2
 
-*Content under construction.*
+B2 is a high-level build system, developed to use with Boost, designed to make it easy to build pass:[C++] projects. Building projects using B2 often involves no more than a single command line. This section introduces what library developers need to know about preparing a library for B2.
 
-https://www.bfgroup.xyz/b2/[B2]
\ No newline at end of file
+== Configuration Files
```

Comment:
In a way, this section describes more what build systems are than what B2 is. Unless this content was specifically required by Rene, I'd try to improve on that.

Reply:
Added more links into B2 docs - but not trying to repeat information that is already in those docs - the purpose of this section is simply to educate a new lib developer on what is needed from them (Jamfiles) - @grafikrobot - any input on this section?